src/resources/stubs/migrations/2014_02_04_000000_create_country_state_city_table.stub

```diff
@@ -17,6 +17,8 @@ class CreateCountryStateCityTable extends Migration
         Schema::create('countries', function (Blueprint $table) {
             $table->bigIncrements('id');
```

Comment:
also, add the Country ISO code with 2 digit and 3 digit

Reply:
Alright, I will do that